engine/runtime/src/main/java/org/enso/interpreter/runtime/data/EnsoMultiValue.java

```diff
@@ -432,9 +427,17 @@ public abstract static class CastToNode extends Node {
      *
      * @param type the requested t
      * @param mv a multi value
+     * @param reorderOnly allow (modified) {@link EnsoMultiValue} to be returned
```

Comment:
what happens if `reorderOnly` is false? I really appreciate the doc, but it would be great if it were a bit more detailed.

Reply:
Done in f5fbcf4097